.github/workflows/yamllint.yml

```
@@ -1,11 +1,14 @@
+---
+
 name: Yaml Lint
-on: [push,pull_request]
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
 jobs:
   lintAllTheThings:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - name: yaml-lint
-      uses: ibiqlik/action-yamllint@v3
-      with:
-        config_file: .yamllint.yml
\ No newline at end of file
+      - uses: actions/checkout@v1
+      - name: yaml-lint
+        uses: ibiqlik/action-yamllint@v3
+        with:
+          config_file: .yamllint.yml
```

```suggestion
          config_file: .yamllint
```

Otherwise this leads to a problem and `yamllint` cannot run.